From 9d390495e55bd4cd55c9f831227d6ae958c92545 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Mon, 13 Jan 2014 12:13:27 +0000
Subject: Cope with Kubuntu setting GRUB_DISTRIBUTOR

This is not a very good approach, and certainly not sanely upstreamable;
we probably need to split GRUB_DISTRIBUTOR into a couple of different
variables.

Repurposed for use by elementary OS by Cody Garver <cody@elementary.io>

Bug-Ubuntu: https://bugs.launchpad.net/bugs/1242417
Forwarded: not-needed
Last-Update: 2013-12-25

Patch-Name: install_efi_ubuntu_flavours.patch
---
 util/grub-install.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/util/grub-install.c b/util/grub-install.c
index 11940d6e6..e1a0202da 100644
--- a/util/grub-install.c
+++ b/util/grub-install.c
@@ -1101,6 +1101,8 @@ main (int argc, char *argv[])
       */
       char *t;
       efi_distributor = bootloader_id;
+      if (strcmp (efi_distributor, "elementary") == 0)
+	efi_distributor = "ubuntu";
       switch (platform)
 	{
 	case GRUB_INSTALL_PLATFORM_I386_EFI:
